src/main/java/mate/academy/carsharing/service/payment/PaymentServiceImpl.java

```diff
@@ -0,0 +1,128 @@
+package mate.academy.carsharing.service.payment;
+
+import com.stripe.model.checkout.Session;
+import jakarta.transaction.Transactional;
+import java.math.BigDecimal;
+import java.time.temporal.ChronoUnit;
+import lombok.RequiredArgsConstructor;
+import mate.academy.carsharing.dto.payment.PaymentRequestDto;
+import mate.academy.carsharing.dto.payment.PaymentResponseDto;
+import mate.academy.carsharing.exceptions.EntityNotFoundException;
+import mate.academy.carsharing.exceptions.PaymentException;
+import mate.academy.carsharing.mapper.PaymentMapper;
+import mate.academy.carsharing.model.Payment;
+import mate.academy.carsharing.model.Rental;
+import mate.academy.carsharing.repository.PaymentRepository;
+import mate.academy.carsharing.repository.RentalRepository;
+import mate.academy.carsharing.service.payment.stripe.StripePaymentService;
+import mate.academy.carsharing.service.telegram.NotificationMessageService;
+import mate.academy.carsharing.service.telegram.NotificationService;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class PaymentServiceImpl implements PaymentService {
+    private static final BigDecimal DAILY_FINE_MULTIPLIER = BigDecimal.valueOf(1.4);
+    private final PaymentRepository paymentRepository;
+    private final StripePaymentService stripePaymentService;
+    private final PaymentMapper paymentMapper;
+    private final RentalRepository rentalRepository;
+    private final NotificationService notificationService;
+    private final NotificationMessageService notificationMessageService;
+
+    @Override
+    public Page<PaymentResponseDto> getWithUserId(Pageable pageable, Long userId) {
+        return paymentRepository.findAllByRental_UserId(pageable, userId)
+                .map(paymentMapper::toDto);
+
```

Comment:
remove redundant empty line
```suggestion
    public Page<PaymentResponseDto> getWithUserId(Pageable pageable, Long userId) {
        return paymentRepository.findAllByRental_UserId(pageable, userId)
                .map(paymentMapper::toDto);
```